starboard/rdk/launcher.py

```diff
@@ -137,7 +137,7 @@ def _InitPexpectCommands(self):
     self.out_directory = self.out_directory.rstrip('/')
 
     rdk_user_hostname = f'{Launcher._RDK_USERNAME}@{self.device_id}'
-    rdk_test_dir = '/usr/share/content/data/app'
```

Comment:
I prefer if keep `/usr/share/content/data/app` as `rdk_test_dir`. That is where the tests are actually run from. The fact that it's a symlink to the `/data` partition is separate.

Reply:
Fixed in https://github.com/youtube/cobalt/pull/1697